react-components/src/components/RevealContainer/RevealResourcesCountContext.tsx

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement, createContext, useContext, useState, type ReactNode } from 'react';
+
+type RevealResourcesCountContent = {
```

Comment:
typo
```suggestion
type RevealResourcesCountContext = {
```

Reply:
`RevealResourcesCountContext` is already declared in the next line, will rename it something else!